scripts/docker/docker_build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -x
+
+docker build -t vast-io/vast:latest .
+read -r VERSION<../../VERSION
+docker tag vast-io/vast:latest vast-io/vast:${VERSION}
+# Export the image to gzipped archives
+docker save vast-io/vast:${VERSION} | gzip > vast-${VERSION}-docker.tar.gz
```

Comment:
Why do we need this file here in addition to the top-level directory?

Reply:
It's a different script. The stuff in `scripts/docker` is intended for automatic creation of docker images from the build artifacts on Jenkins. The top-level one can be used to create images from scratch.

Reply:
I'm thinking of removing `scripts/docker`, since we won't be enabling automatic image creation for VAST, if I remember correctly.

Reply:
> It's a different script.

Okay.

> I'm thinking of removing `scripts/docker`, since we won't be enabling automatic image creation for VAST, if I remember correctly.

But if it's Jenkins-related, it's probably a good idea to have it in the repo as well, no? Then it would live along with the `Jenkinsfile` and can be kept in sync.

Reply:
We don't need to build a VAST docker image in Jenkins in order to build a Tenzir image. I don't think we need to build a VAST image every time Jenkins runs for the master branch. If we don't use it, why should we introduce it in the first place?

Reply:
> If we don't use it, why should we introduce it in the first place?

I don't have a strong opinion when it comes to removing the script. We can do that if you prefer it. I just thought that having it in the repo is quite a natural thing, because it's easier to update together.

Reply:
I decided to drop this. We won't build docker images from Jenkins artifacts, so this does not give us any value.